### ISSUE TITLE
How to train a new style

### ISSUE THREAD

**Issue by wjjjjyourFA** [naoto0804/pytorch-AdaIN#44]
Hi.
I wonder how to train a new style.

I put a image in ./input/content and another in ./input/style, using "vgg_normalized.pth"
CMD:`python train.py --content_dir input/content --style_dir input/style --batch_size 4 --max_iter 160`
Though I got a xx.pth.tar in ./experiments, I use it "test.py", getting a gray image result. 

source src
![Image-001628821383431](https://user-images.githubusercontent.com/42645285/129871251-4a20d5cb-6bd6-4418-a676-3728d32a588f.jpg)

result src
![Image-001628821383431_stylized_004778408333303](https://user-images.githubusercontent.com/42645285/129871302-395bb666-122c-47fc-8ede-f6d16d63e45d.jpg)

**Comment by naoto0804** [comment]
I guess `max_iter` is too small. Please try the default parameters, or keep an eye on the convergence of loss curves to make sure you train the model enough.
